tests/keras/test_custom_op.py

```diff
@@ -0,0 +1,57 @@
+import sys
```

Comment:
Don't we need some tests for Core ML's customized operator?

Reply:
Yes, will do in the next PR.